jekyll/_cci2/server-3-operator-backup-and-restore.adoc

```diff
@@ -366,6 +370,121 @@ velero-68788b675c-dm2s7   1/1     Running   0          2m
 
 As restic is a daemonset, there should be one pod for each node in your Kubernetes cluster.
 
+////
+
+* S3-COMPATIBLE SETUP *
+
+////
+== Server 3.x backups with S3 Compatible Storage
+
+The following steps will assume you're using S3 compatible object storage, but not necessarily AWS S3, for your backups.
+It is also assumed you have met the <<s3-compatible-storage-prerequisites, prerequisites>>.
+
+These instructions were sourced from the Velero documentation https://velero.io/docs/v1.6/contributions/minio/[here].
+
+=== Step 1 - Configure `mc` client
+
+To start, configure https://docs.min.io/minio/baremetal/reference/minio-cli/minio-mc.html[`mc`] to connect to your storage
+provider:
+
+[source,bash]
+----
+# Alias can be any name as long as you use the same value in subsequent commands
+export ALIAS=my-provider
+// Can we clarify what these 3 params are exactly
+mc alias set $ALIAS <YOUR_ENDPOINT> <YOUR_ACCESS_KEY_ID> <YOUR_SECRET_ACCESS_KEY>
+----
+
+You can verify your client is correctly configured by running `mc ls my-provider` and you should see the buckets in your provider enumerated in the output.
+
+=== Step 2 - Create a bucket
+
+Create a bucket for your backups. It is important that a new bucket is used, as Velero cannot use a preexisting bucket with other content.
+
+[source, bash]
+----
+mc mb ${ALIAS}/<YOUR_BUCKET>
+----
+
+=== Set 3 - Create a user and policy
+
+Next, create a user and policy for Velero to access your bucket.
+
+[source, bash]
+----
+# Create user
+mc admin user add $ALIAS <YOUR_VELERO_ACCESS_KEY_ID> <YOUR_VELERO_SECRET_ACCESS_KEY>
```

Comment:
Ah, I see why this is a little confusing. We state <YOUR_VELERO_ACCESS_KEY_ID> here. The intention is that is the the key id for Velero to access Minio, but it reads like the _your_ key to access Velero. @annabaker do you have any suggestions on a more clear variable name?

Reply:
I would add something like "<YOUR_MINIO_ACCESS_KEY_ID>" with some context around, the minio access key to be used by Velero.

Reply:
I've added suggestions to hopefully make things clearer on your suggestions!